editor/src/core/model/element-metadata-utils.ts

```diff
@@ -1042,26 +1055,14 @@ export const MetadataUtils = {
 
       // Checking if our elements support children should prevent us from ending up with the
       // internals of draft-js showing up underneath Text elements.
+      // TODO is data-utopia-do-not-traverse still relevant?
```

Comment:
This prop mainly existed to prevent us digging into DraftJS, but I'm pretty sure that's no longer an issue.